apps/aecore/lib/aecore/wallet/worker.ex

```diff
@@ -0,0 +1,92 @@
+defmodule Aecore.Wallet.Worker do
+  @defmodule """
+  Module for handling the creation of a Wallet file
+  """
+
+  use GenServer
+
+  alias Aewallet.Wallet
+
+  @typedoc "Options for network"
+  @type opts :: :mainnet | :testnet
+
+  @aewallet_dir Application.get_env(:aecore, :aewallet)[:path]
+  @aewallet_pass Application.get_env(:aecore, :aewallet)[:pass]
```

Comment:
If I'm not mistaken this will get evaluated compile time, not runtime!

Reply:
Will make two function for calling them at run time